src/hotspot/share/prims/jvmtiRedefineClasses.cpp

```diff
@@ -932,6 +932,11 @@ jvmtiError VM_RedefineClasses::compare_and_normalize_class_versions(
       (the_class->super() == NULL || scratch_class->super() == NULL ||
        the_class->super()->name() !=
        scratch_class->super()->name())) {
+    log_trace(redefine, class, normalize)
```

Comment:
I think these would be useful to users so that these should be made log_info(redefine). All the other redefinition logging is useful for developers in debugging redefinition, and a user wouldn't really be able to follow the output unless they are very knowledgeable.  Making this log_info(redefine) would let them see this without the noise of the other output.

Reply:
I've made them log_trace to be consistent with already existing logging introduced for nestmates related errors. Do you want to make all of them log_info in such case?

Reply:
I see.  I think so, yes.  Since you're returning an error from redefinition, a higher level logging seems warranted.

Reply:
I've updated logging to use log_info where error is returned.